extra_data/reader.py

```diff
@@ -1633,6 +1633,107 @@ def __contains__(self, aliased_item):
         except KeyError:
             return False
 
+    def __repr__(self):
+        """
+        Pretty-print all the aliases.
+        """
+        RED = "\033[91m"
+        END_COLOR = "\033[0m"
+
+        # Check if the alias exists
+        def alias_exists(alias):
```

Comment:
Why not use `AliasIndexer.__contains__` added in https://github.com/European-XFEL/EXtra-data/pull/398?